http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriberSpec.scala

```diff
@@ -1606,6 +1622,296 @@ class LeoPubsubMessageSubscriberSpec
     res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
+  it should "create azure vm properly" in isolatedDbTest {
```

Comment:
shall we have most of these tests in `AzureInterpreterSpec`...the logic in pubsub is pretty straightforward...

Reply:
Sure. There are two tests that aim to verify a top-level error results in pubsub correctly updating the cluster to error that I'll leave in this spec, but the rest can just use the interp directly.

Reply:
cool..yea..mainly I think this spec is getting a bit out of hand. And unit tests should be closer to what's being tested if possible